fastlane/lanes/test.rb

```diff
@@ -19,40 +17,15 @@
 
     gradle(tasks: ["WordPress:assemble#{app.to_s.capitalize}VanillaDebug", "WordPress:assemble#{app.to_s.capitalize}VanillaDebugAndroidTest"])
 
-    # Run the instrumented tests in Firebase Test Lab
-    firebase_login(
-      key_file: GOOGLE_FIREBASE_SECRETS_PATH
-    )
-
-    apk_dir = File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'build', 'outputs', 'apk')
-
-    test_succeeded = android_firebase_test(
-      project_id: firebase_secret(name: 'project_id'),
-      key_file: GOOGLE_FIREBASE_SECRETS_PATH,
-      model: 'Pixel2.arm',
-      version: 30,
-      test_apk_path: File.join(apk_dir, 'androidTest', "#{app}Vanilla", 'debug', "org.wordpress.android-#{app}-vanilla-debug-androidTest.apk"),
-      apk_path: File.join(apk_dir, "#{app}Vanilla", 'debug', "org.wordpress.android-#{app}-vanilla-debug.apk"),
-      test_targets: 'notPackage org.wordpress.android.ui.screenshots',
-      results_output_dir: File.join(PROJECT_ROOT_FOLDER, 'build', 'instrumented-tests'),
-      crash_on_test_failure: false
-    )
-
     annotation_ctx = "firebase-test-#{app}-vanilla-debug"
-    if test_succeeded
+    begin
+      gradle(task: "runFlank#{app.to_s.capitalize}")
       sh("buildkite-agent annotation remove --context '#{annotation_ctx}' || true") if is_ci?
-    else
-      details_url = lane_context[SharedValues::FIREBASE_TEST_MORE_DETAILS_URL]
-      message = "Firebase Tests failed. Failure details can be seen [here in Firebase Console](#{details_url})"
+    rescue
+      details_url = sh('jq ".[].webLink" ../build/instrumented-tests/matrix_ids.json -r')
```

Comment:
For the `sh` call itself, I'd suggest to break down the command and its arguments (in principle for easier reading, but also for not leaving room for  command injections, issues with spaces in arguments, unwanted interpolation, etc):

```suggestion
      details_url = sh('jq', '.[].webLink', '../build/instrumented-tests/matrix_ids.json', '-r')
```

Reply:
Sure, thanks. Addressed in e7d901b8f318e810a92fcb0c3745600ccf5fe9ec